suite/tests/CMakeLists.txt

```diff
@@ -2151,6 +2151,9 @@ if (CLIENT_INTERFACE)
       # For the subtest "targeting upper 2GB of low 4GB" we move -vm_base out of the
       # way.  We also raise the checklevel so we get memory filling for client allocs.
       "-no_vm_base_near_app -vm_base 0x120000000 -checklevel 3" "")
+    # A second test for i#4335's OOM despite a reset.  We leave checklevel for CI speed.
+    tobuild_ci(client.alloc-noreset client-interface/alloc.c ""
+      "-no_enable_reset -no_vm_base_near_app -vm_base 0x120000000" "")
```

Comment:
I see some tests that explicitly turn on the `enable_reset` option. Is this option on or off by default?